<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MemberViewDialog</class>
 <widget class="QDialog" name="MemberViewDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>692</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="windowIcon">
   <iconset resource="img.qrc">
    <normaloff>:/src/src/0b85636ca6384399edbae4cc3d344937.png</normaloff>:/src/src/0b85636ca6384399edbae4cc3d344937.png</iconset>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="2" column="0">
      <widget class="QLabel" name="phone_label">
       <property name="text">
        <string>Phone Number</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0" alignment="Qt::AlignTop">
      <widget class="QLabel" name="first_name_label">
       <property name="text">
        <string>Name</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="address_label">
       <property name="text">
        <string>Home Address</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2" colspan="2">
      <widget class="QLineEdit" name="phone_line_edit">
       <property name="inputMask">
        <string/>
       </property>
      </widget>
     </item>
     <item row="0" column="2" colspan="2">
      <widget class="QLineEdit" name="first_name_line_edit"/>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="penalty_label">
       <property name="text">
        <string>Penalty</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2" colspan="2">
      <widget class="QLineEdit" name="last_name_line_edit">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="email_label">
       <property name="text">
        <string>Email</string>
       </property>
      </widget>
     </item>
     <item row="7" column="0" alignment="Qt::AlignTop">
      <widget class="QLabel" name="borrowed_label">
       <property name="text">
        <string>Borrowed Books</string>
       </property>
      </widget>
     </item>
     <item row="8" column="0" colspan="4">
      <widget class="QLabel" name="msg_label">
       <property name="text">
        <string>User Found</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="last_name_label">
       <property name="text">
        <string>National ID</string>
       </property>
      </widget>
     </item>
     <item row="4" column="2" colspan="2">
      <widget class="QLineEdit" name="address_line_edit"/>
     </item>
     <item row="3" column="2" colspan="2">
      <widget class="QLineEdit" name="email_line_edit"/>
     </item>
     <item row="7" column="3">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item alignment="Qt::AlignTop">
        <widget class="QPushButton" name="return_button">
         <property name="text">
          <string>Return</string>
         </property>
         <property name="icon">
          <iconset resource="img.qrc">
           <normaloff>:/src/src/return.png</normaloff>:/src/src/return.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="renew_book_button">
         <property name="text">
          <string>Renew</string>
         </property>
         <property name="icon">
          <iconset resource="img.qrc">
           <normaloff>:/src/src/browser.png</normaloff>:/src/src/browser.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item row="6" column="0" colspan="4">
      <widget class="Line" name="line_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="5" column="2" colspan="2">
      <widget class="QLineEdit" name="penalty_line_edit">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="7" column="2">
      <widget class="QTableWidget" name="borrow_table">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <property name="columnCount">
        <number>4</number>
       </property>
       <column>
        <property name="text">
         <string>Loan Index</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Title</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Borrow Date</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Times Renewed</string>
        </property>
       </column>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="lend_button">
       <property name="text">
        <string>Lend Book</string>
       </property>
       <property name="icon">
        <iconset resource="img.qrc">
         <normaloff>:/src/src/book.png</normaloff>:/src/src/book.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="update_button">
       <property name="text">
        <string>Update Info</string>
       </property>
       <property name="icon">
        <iconset resource="img.qrc">
         <normaloff>:/src/src/replace.png</normaloff>:/src/src/replace.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="renew_member_button">
       <property name="text">
        <string>Renew Membership</string>
       </property>
       <property name="icon">
        <iconset resource="img.qrc">
         <normaloff>:/src/src/add-user.png</normaloff>:/src/src/add-user.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="add_transaction_button">
       <property name="text">
        <string>Add Payment</string>
       </property>
       <property name="icon">
        <iconset resource="img.qrc">
         <normaloff>:/src/src/credit-card.png</normaloff>:/src/src/credit-card.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="add_penalty_button">
       <property name="text">
        <string>Add Penalty</string>
       </property>
       <property name="icon">
        <iconset resource="img.qrc">
         <normaloff>:/src/src/add (1).png</normaloff>:/src/src/add (1).png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="close_button">
       <property name="text">
        <string>Close</string>
       </property>
       <property name="icon">
        <iconset resource="img.qrc">
         <normaloff>:/src/src/cancel.png</normaloff>:/src/src/cancel.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>first_name_line_edit</tabstop>
  <tabstop>last_name_line_edit</tabstop>
  <tabstop>phone_line_edit</tabstop>
  <tabstop>email_line_edit</tabstop>
  <tabstop>address_line_edit</tabstop>
  <tabstop>penalty_line_edit</tabstop>
  <tabstop>return_button</tabstop>
  <tabstop>renew_book_button</tabstop>
  <tabstop>lend_button</tabstop>
  <tabstop>update_button</tabstop>
  <tabstop>renew_member_button</tabstop>
  <tabstop>add_transaction_button</tabstop>
  <tabstop>add_penalty_button</tabstop>
  <tabstop>close_button</tabstop>
 </tabstops>
 <resources>
  <include location="img.qrc"/>
 </resources>
 <connections/>
</ui>
